rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -482,45 +483,46 @@ namespace rocal{
-        //     py::return_value_policy::reference);
-        // m.def("Caffe2_ImageDecoderShard", &rocalJpegCaffe2LMDBRecordSourceSingleShard,"Reads file from the source given and decodes it according to the shard id and number of shards",
-        //     py::return_value_policy::reference);
-        // m.def("Caffe2_ImageDecoderPartialShard", &rocalJpegCaffe2LMDBRecordSourcePartialSingleShard);
-        // m.def("FusedDecoderCrop", &rocalFusedJpegCrop,"Reads file from the source and decodes them partially to output random crops",
-        //     py::return_value_policy::reference);
-        // m.def("FusedDecoderCropShard", &rocalFusedJpegCropSingleShard,"Reads file from the source and decodes them partially to output random crops",
-        //     py::return_value_policy::reference);
-        // m.def("TF_ImageDecoderRaw", &rocalRawTFRecordSource,"Reads file from the source given and decodes it according to the policy only for TFRecords",
-        //       py::return_value_policy::reference);
-        // m.def("Cifar10Decoder", &rocalRawCIFAR10Source,"Reads file from the source given and decodes it according to the policy only for TFRecords",
-        //       py::return_value_policy::reference);
+        m.def("cocoImageDecoder", &rocalJpegCOCOFileSource,"Reads file from the source given and decodes it according to the policy",
+            py::return_value_policy::reference);
+        m.def("cocoImageDecoderShard", &rocalJpegCOCOFileSourceSingleShard,"Reads file from the source given and decodes it according to the shard id and number of shards",
+            py::return_value_policy::reference);
+        m.def("tfImageDecoder", &rocalJpegTFRecordSource,"Reads file from the source given and decodes it according to the policy only for TFRecords",
+            py::return_value_policy::reference);
+        m.def("caffeImageDecoder", &rocalJpegCaffeLMDBRecordSource,"Reads file from the source given and decodes it according to the policy",
+            py::return_value_policy::reference);
+        m.def("caffeImageDecoderShard", &rocalJpegCaffeLMDBRecordSourceSingleShard, "Reads file from the source given and decodes it according to the shard id and number of shards",
+            py::return_value_policy::reference);
+        m.def("caffeImageDecoderPartialShard", &rocalJpegCaffeLMDBRecordSourcePartialSingleShard, py::return_value_policy::reference);
+        m.def("caffe2ImageDecoder", &rocalJpegCaffe2LMDBRecordSource,"Reads file from the source given and decodes it according to the policy",
+            py::return_value_policy::reference);
+        m.def("caffe2ImageDecoderShard", &rocalJpegCaffe2LMDBRecordSourceSingleShard,"Reads file from the source given and decodes it according to the shard id and number of shards",
+            py::return_value_policy::reference);
+        m.def("caffe2ImageDecoderPartialShard", &rocalJpegCaffe2LMDBRecordSourcePartialSingleShard, py::return_value_policy::reference);
```

Comment:
In caffeImageDecoderPartialShard and caffe2ImageDecoderPartialShard move the return value policy to next line similar to others and add description